source/plugins/ruby/in_cadvisor_perf.rb

```diff
@@ -10,6 +10,7 @@ class CAdvisor_Perf_Input < Input
     if !@@os_type.nil? && !@@os_type.empty? && @@os_type.strip.casecmp("windows") == 0
       @@isWindows = true
     end
+    @@totalPerfCount = 0
```

Comment:
same here .. instance variable would be good enough for totalPerfCount since this being shared other instances of the plugins, right?